### DISCUSSION TITLE
Where do you find the test branch?

### DISCUSSION THREAD

**Discussion by rickygrove** [mdotstrange/NightmarePuppeteerPublic#380] [General]
I'd like to start using the new fbx import, but I don't know where to look for the test branch. Eeek!

**Comment by mdotstrange** [comment]
Hey Ricky sorry I missed this post! The instructions are on a Steam news post on the NP page [here](https://steamcommunity.com/games/1355310/announcements/detail/3024709067886113771) https://steamcommunity.com/games/1355310/announcements/detail/3024709067886113771

**Comment by rickygrove** [comment]
I figured it out. thanks for double checking though.

Ricy
